sdks/how-to/how-to-create-a-data-sample-with-hierarchical-information.md

```diff
@@ -0,0 +1,393 @@
+---
+slug: how-to-create-a-data-sample-with-hierarchical-information
+description: Learn how to create a data sample with hierarchical information.
+tags:
+- DataSample
+
+---
+
+# How to create a data sample with hierarchical information
+
+In this section, we will learn how to create data samples with complex nested objects. DataSamples are used to store
+data that is not part of the patient's medical record. In some circumstances, the data sample may contain complex nested
+objects.
```

Comment:
Please update the "patient medical record" part also in the other data sample how to